kgforge/specializations/stores/bluebrain_nexus.py

```diff
@@ -1018,6 +1043,63 @@ def format_message(msg):
         return format_message(str(error))
 
 
+def build_sparql_query_statements(context: Context, *conditions) -> Tuple[List, List]:
```

Comment:
This was ported in [SPARQLQueryBuilder](https://github.com/BlueBrain/nexus-forge/blob/master/kgforge/core/commons/sparql_query_builder.py).

Reply:
Ah, yes, I think it got in by mistake when I rebased.